examples/lightgbm_simple.py

```diff
@@ -37,18 +40,16 @@ def objective(trial):
         'objective': 'binary',
         'metric': 'binary_logloss',
         'verbosity': -1,
-        'boosting_type': trial.suggest_categorical('boosting', ['gbdt', 'dart', 'goss']),
-        'num_leaves': trial.suggest_int('num_leaves', 10, 1000),
-        'learning_rate': trial.suggest_loguniform('learning_rate', 1e-8, 1.0)
+        'boosting_type': 'gbdt',
+        'lambda_l1': trial.suggest_loguniform('lambda_l1', 1e-8, 10.0),
+        'lambda_l2': trial.suggest_loguniform('lambda_l2', 1e-8, 10.0),
+        'num_leaves': trial.suggest_int('num_leaves', 2, 256),
+        'feature_fraction': min(trial.suggest_uniform('feature_fraction', 0.4, 1.0 + EPS), 1.0),
+        'bagging_fraction': min(trial.suggest_uniform('bagging_fraction', 0.4, 1.0 + EPS), 1.0),
+        'bagging_freq': trial.suggest_int('bagging_freq', 1, 7),
+        'min_child_samples': int(trial.suggest_uniform('min_child_samples', 5, 100 + EPS)),
```

Comment:
How about using `suggest_int` as follows?
```suggestion
        'min_child_samples': trial.suggest_int('min_child_samples', 5, 100),
```

Reply:
Good! I'll fix it.

Reply:
Fixed: d22a7ce